airbrakes/data_handling/data_processor.py

```diff
@@ -298,4 +267,4 @@ def _calculate_time_differences(self) -> npt.NDArray[np.float64]:
         # We are converting from ns to s, since we don't want to have a velocity in m/ns^2
         # We are using the last data point to calculate the time difference  between the last data point from the
         # previous loop, and the first data point from the current loop
-        return np.diff([data_packet.timestamp for data_packet in [self._last_data_packet, *self._data_packets]]) * 1e-9
+        return np.diff([data_packet.timestamp * 1e-9 for data_packet in [self._last_data_packet, *self._data_packets]])
```

Comment:
we should change this to using convertNanoseconds or whatever in utils

Reply:
i'm confused, we want it in seconds... not nanoseconds